scripts/js/lib/api/updateLinks.ts

```diff
@@ -166,7 +172,8 @@ export function relativizeLink(link: Link): Link | undefined {
 
   const newText = link.url === link.text ? url : undefined;
   const newPrefix = priorPrefixToNewPrefix.get(priorPrefix)!;
-  return { url: `${newPrefix}/${url}`, text: newText };
+  const relativeUrl = removePrefix(join(newPrefix, url), "/");
+  return { url: `/${relativeUrl}`, text: newText };
```

Comment:
I had to change the logic here to fix some cases of double-slashes (e.g. `/docs//guides/page`).

Reply:
Sg. Do your tests capture that edge case?

Reply:
Yep, the tests actually uncovered it